app/controllers/feedback_controller.rb

```diff
@@ -22,4 +22,14 @@ def create
   def contact_params
     params.require(:contact).permit(:name, :email, :message)
   end
+
+  def set_layout
+    if current_user.nil?
+      "application"
+    elsif current_user.staff_account
```

Comment:
we are deprecating the staff_account very soon. can you instead use the Rolify method `has_role?` and pass in the role you want to check, and the current organization, since roles are scoped to organizatoin on a user. 

so something like 
```
if current_user.has_role?(:admin, ActsAsTenant.current_tenant) ||
  current_user.has_role?(:super_admin, ActsAsTenant.current_tenant)
```

Reply:
It might be easier to check for the `adopter` role instead and set the fallback to the staff dashboard...less code